src/plugins/uk.ac.kcl.mdeoptimise.ui/src/uk/ac/kcl/ui/quickfix/MDEOptimiseQuickfixProvider.xtend

```diff
@@ -12,13 +26,29 @@ import org.eclipse.xtext.ui.editor.quickfix.DefaultQuickfixProvider
  */
 class MDEOptimiseQuickfixProvider extends DefaultQuickfixProvider {
 
-//	@Fix(MDEOptimiseValidator.INVALID_NAME)
-//	def capitalizeName(Issue issue, IssueResolutionAcceptor acceptor) {
-//		acceptor.accept(issue, 'Capitalize name', 'Capitalize the name.', 'upcase.png') [
-//			context |
-//			val xtextDocument = context.xtextDocument
-//			val firstLetter = xtextDocument.get(issue.offset, 1)
-//			xtextDocument.replace(issue.offset, 1, firstLetter.toUpperCase)
-//		]
-//	}
-}
+	@Inject
+	private IJavaProjectProvider projectProvider;
+	
+	@Fix(MDEOptimiseValidatorIssues.MDEO_LIB_NOT_ON_CLASSPATH)
+	def putMDEOOnClasspath(Issue issue, IssueResolutionAcceptor acceptor) {
+		
```

Comment:
I likes it. I hadn't used quick fixes in my Xtext languages before, but this seems nicely compact :-)